src/ServiceControlInstaller.Engine/Instances/Transports.cs

```diff
@@ -17,37 +17,44 @@ public static TransportInfo FindByName(string name)
             {
                 Name = "AzureServiceBus",
                 TypeName = "NServiceBus.AzureServiceBusTransport, NServiceBus.Azure.Transports.WindowsAzureServiceBus",
-                MatchOn = "NServiceBus.AzureServiceBus",
+                MatchOn = "NServiceBus.AzureServiceBusTransport",
                 SampleConnectionString = @"Endpoint=sb://[namespace].servicebus.windows.net; SharedSecretIssuer=<owner>;SharedSecretValue=<someSecret>"
 
             },
             new TransportInfo
             {
                 Name = "AzureStorageQueue",
                 TypeName = "NServiceBus.AzureStorageQueueTransport, NServiceBus.Azure.Transports.WindowsAzureStorageQueues",
-                MatchOn  = "NServiceBus.AzureStorageQueue",
+                MatchOn  = "NServiceBus.AzureStorageQueueTransport",
                 SampleConnectionString = @"DefaultEndpointsProtocol=[http|https];AccountName=<MyAccountName>;AccountKey=<MyAccountKey>"
             },
             new TransportInfo
             {
                 Name = "MSMQ",
                 TypeName = "NServiceBus.MsmqTransport, NServiceBus.Core", 
-                MatchOn = "NServiceBus.Msmq",
+                MatchOn = "NServiceBus.MsmqTransport",
                 SampleConnectionString = @String.Empty,
                 Default = true
             },
             new TransportInfo
             {
                 Name = "SQLServer",
                 TypeName = "NServiceBus.SqlServerTransport, NServiceBus.Transports.SQLServer",
-                MatchOn = "NServiceBus.SqlServer",
+                MatchOn = "NServiceBus.SqlServerTransport",
+                SampleConnectionString = @"Data Source=<SQLInstance>;Initial Catalog=nservicebus;Integrated Security=True"
+            },
+            new TransportInfo
+            {
+                Name = "SQLServer with DTC",
+                TypeName = "NServiceBus.SqlServerWithDTCTransport, NServiceBus.Transports.SqlServerWithDTC",
```

Comment:
@johnsimons 

The TypeName is what ends up in the app config in ""ServiceControl/TransportType" so it that what you intents here?